plugin/utils/flag.py

```diff
@@ -249,14 +249,14 @@ def build(self):
         "-dependency-dot",
         "-dependency-file",
         "-F",
-        "-fmodules-cache-path",
```

Comment:
Why is this removed?

Reply:
It should be used with equality (-fmodules-cache-path=<directory>)

Reply:
It is already in the list of separable flags. It will be used with `=` sign. Or does this not work for you?

Reply:
It hadn't worked in my case, so I removed that line.